src/test/java/world/bentobox/bentobox/listeners/flags/protection/PhysicalInteractionListenerTest.java

```diff
@@ -98,7 +100,7 @@ public void setUp() {
         ItemFactory itemFactory = mock(ItemFactory.class);
         when(server.getItemFactory()).thenReturn(itemFactory);
 
-        PowerMockito.mockStatic(Bukkit.class);
+        PowerMockito.mockStatic(Bukkit.class, Mockito.RETURNS_MOCKS);
```

Comment:
This looks interesting. I haven't seen this option before.

Reply:
Well, thanks StackOverflow for that one. They provided something like `PowerMockito.mockStatic(clazz, Mockito.DEEP_SUB...);` and, out of rage with that thing not working, I tried this RETURNS_MOCKS. That works like a charm 😄